source/geod24/concurrency.d

```diff
@@ -1446,3 +1502,665 @@ private:
     self.receiveOnly!(bool);
     assert(x[0] == 5);
 }
+
+/***********************************************************************
+
+    A golang style channel implementation with buffered and unbuffered
+    operation modes
+
+    Intended to be used between Fibers
+
+***********************************************************************/
+
+final public class Channel (T)
```

Comment:
`T` needs `Parameter` documentation. Are there any constraints here? T can be anything?

Reply:
Are there any examples of this? Cant find a special keyword for template documentation in https://dlang.org/spec/ddoc.html

Reply:
It's not special, it is simply a parameter for the documentation generator.